clang/unittests/Format/FormatTest.cpp

```diff
@@ -20323,6 +20323,61 @@ TEST_F(FormatTest, AlignConsecutiveDeclarations) {
                "void f2(void);\n"
                "size_t f3(void);",
                Alignment);
+
+  Alignment.AlignConsecutiveDeclarations.AlignFreeVariableDeclarations = false;
+  verifyFormat("int func() { //\n"
+               "  int local1;\n"
+               "  unsigned local2;\n"
```

Comment:
I think you should add a function pointer to the free and member variables, to show they don't override the setting of `AlignFunctionPointers`.